sdk/appconfiguration/azure-appconfiguration/azure/appconfiguration/_utils.py

```diff
@@ -65,3 +77,26 @@ def get_label_filter(*args: Optional[str], **kwargs: Any) -> Tuple[Optional[str]
                 "'label_filter'"
             )
     return label_filter or kwargs.pop("label_filter", None), kwargs
+
+
+def get_default_scope(endpoint: str, audience: Optional[str] = None) -> str:
+    """
+    Gets the default scope for the given endpoint.
+
+    :param endpoint: The endpoint to get the default scope for.
+    :type endpoint: str
+    :param audience: The audience to use for the scope. If not provided, determines audience based on endpoint.
+    :type audience: str or None
+    :return: The default scope for the given endpoint.
+    :rtype: str
+    """
+    # Normalize endpoint by stripping trailing slashes for suffix checks
+    normalized_endpoint = endpoint.rstrip('/')
```

Comment:
case insensitive comparison is needed here

Reply:
Do we? The version the Java SDK wrote isn't: https://github.com/Azure/azure-sdk-for-java/blob/854242443d2493f56f2fb6762137a3b7d1f4c625/sdk/appconfiguration/azure-data-appconfiguration/src/main/java/com/azure/data/appconfiguration/ConfigurationClientBuilder.java#L572

JS isn't: https://github.com/Azure/azure-sdk-for-js/blob/7467280322bc325626fd25cf384a755c7b3b337c/sdk/appconfiguration/app-configuration/src/internal/helpers.ts#L488

.Net is: https://github.com/Azure/azure-sdk-for-net/blob/cd673e051186147a777ea27f4df9f3b2d6d27413/sdk/appconfiguration/Azure.Data.AppConfiguration/src/AppConfigurationAudience.cs#L50

@jimmyca15 Thoughts